webapp/packages/plugin-connection-custom/src/CustomConnection/ConnectionFormDialog/ConnectionFormDialogController.ts

```diff
@@ -126,12 +183,25 @@ implements IInitializableController, IDestructibleController {
       config.properties = this.config.properties;
     }
 
+    config.name = config.name?.trim();
+
     return config;
   }
 
+  private validate(config: ConnectionConfig) {
+    const validationStatus: IValidationStatus = { status: true, errorMessage: '' };
+
+    if (!config.name?.length) {
+      validationStatus.errorMessage = "Field 'name' can't be empty";
+    }
+
+    validationStatus.status = !validationStatus.errorMessage;
+    return validationStatus;
+  }
+
   @action
   private setDriverDefaults() {
-    this.config.name = `${this.driver.name} (custom)`;
+    this.config.name = `${this.driver.name}@${this.driver.defaultServer || 'localhost'}:${this.driver.defaultPort || ''}`;
```

Comment:
define name after host and port, so you be able to use `this.config.host` and `this.config.port`